skt/runner.py

```diff
@@ -167,6 +180,15 @@ def _forget_cid(self, cid):
             raise ValueError("Unknown cid type: %s" % cid)
 
     def getverboseresults(self, joblist):
+        """
+        Retrieve verbose results for a list of beaker jobs
+
+        Args:
+            joblist: A list of beaker jobs
+
+        Returns:
+            A dictionary with the jobid as a key and job results as the value
```

Comment:
We need to use "taskspec" here as well, I think, but we'll have to merge this as is.